### PR TITLE
create event post for wics-csc event

### PR DESCRIPTION
**Pull request by claricen**

### CODE REVIEW

File: content/S2016/S16-event-wics-csc.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title: WiCS & CSC Go Outside
+Date: 2016-06-30
+Category: Events
+Tags: social
+Slug: wics-csc-s16
+Author: Clare Ng
+Summary: We're having a bonfire with the CSC! There will be ice cream and marshmellows.
+
+Join us at the Laurel Creek Fire Pit (by EV3) for a night outdoors with fellow people 
+in computer science! There will be ice cream and s'mores! Also fire. And a creek.
```
Comment (evykassirer) [inline content/S2016/S16-event-wics-csc.md:10]: 🔥 🍨

**Comment by evykassirer** [comment]
lgtm but I'll let some other people take a look :)

**Comment by bdamerac** [comment]
lgtm too!

**Comment by evykassirer** [comment]
@SrishtiG want to deploy? or shall I

**Pull request merged**